MediaBrowser.XbmcMetadata/Parsers/BaseNfoParser.cs

```diff
@@ -377,40 +377,42 @@ protected virtual void FetchDataFromXmlNode(XmlReader reader, MetadataResult<T>
 
                 case "watched":
                     {
+                        var val = reader.ReadElementContentAsBoolean();
+
                         if (userData != null)
                         {
-                            userData.Played = reader.ReadElementContentAsBoolean();
+                            userData.Played = val;
                         }
 
                         break;
                     }
 
                 case "playcount":
                     {
-                        if (userData != null)
+                        var val = reader.ReadElementContentAsString();
+                        if (!string.IsNullOrWhiteSpace(val) && userData != null)
                         {
-                            userData.PlayCount = reader.ReadElementContentAsInt();
+                            if (int.TryParse(val.Split(' ')[0], NumberStyles.Integer, UsCulture, out var count))
```

Comment:
Use `IndexOf(' ')` and slice as Span to prevent an allocation

Reply:
`playcount` shouldn't contain spaces at all so don't need to split at all

Reply:
even better